content/en/_index.md

```diff
@@ -197,6 +231,38 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="PHP" >}}
+/**
+ * Lists all Info Types for the Data Loss Prevention (DLP) API.
+ */
+use Google\Cloud\Dlp\V2\DlpServiceClient;
+
+/** Uncomment and populate these variables in your code */
+// $filter = ''; // (Optional) filter to use, empty for ''.
+// $languageCode = ''; // (Optional) language code, empty for 'en-US'.
```

Comment:
Can we avoid commented out variables here? Commented out variables are generally a poor practice because they are not tested or linted at anyway, and can very easily be wrong or outdated.